<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <property name="locale">
   <locale language="Korean" country="SouthKorea"/>
  </property>
  <widget class="QPushButton" name="pb_start">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>40</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_info">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>View</string>
   </property>
   <widget class="QWidget" name="widget_view" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cb_device">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>GPU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CPU</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cb_model">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>VGG-16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MonileNet</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cb_source">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Camera</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Video File</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Device :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Source :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Model :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>271</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Performance</string>
   </property>
   <widget class="QWidget" name="widget_perf" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_stop">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
